src/sentry/demo/data_population.py

```diff
@@ -739,24 +831,80 @@ def generate_alerts(self, project):
     def generate_metric_alert(self, project):
         org = project.organization
         team = Team.objects.filter(organization=org).first()
+        time_interval = self.get_config_var("METRIC_ALERT_INTERVAL")
+        max_days = self.get_config_var("MAX_DAYS")
         alert_rule = create_alert_rule(
             org,
             [project],
             f"High Error Rate - {project.name} ",
             "level:error",
             "count()",
-            10,
+            time_interval,
             AlertRuleThresholdType.ABOVE,
             1,
         )
-        critical_trigger = create_alert_rule_trigger(alert_rule, "critical", 10)
-        warning_trigger = create_alert_rule_trigger(alert_rule, "warning", 7)
-        for trigger in [critical_trigger, warning_trigger]:
-            create_alert_rule_trigger_action(
-                trigger,
-                AlertRuleTriggerAction.Type.EMAIL,
-                AlertRuleTriggerAction.TargetType.TEAM,
-                target_identifier=str(team.id),
+
+        alert_rule.update(
+            date_added=timezone.now() - timedelta(days=max_days),
+            date_modified=timezone.now() - timedelta(days=max_days * 2),
```

Comment:
@roggenkemper there are a bunch of constants that I'm not sure where they come from (ex: the `2` here). Can we make it a variable constant or add a comment explaining what it is?